fire/cli/info.py

```diff
@@ -159,21 +159,32 @@ def info():
 #   Nyeste observation:  1997-07-15 12:00:00
 
 
-def kanonisk_ident(uuid) -> str:
-    try:
-        # TODO: Bør cache både pil, pid og resultater pr UUID, så vi kan reducere opslag
-        pil = firedb.hent_punktinformationtype("IDENT:landsnr")
-        pig = firedb.hent_punktinformationtype("IDENT:GNSS")
-        pid = firedb.hent_punktinformationtype("IDENT:diverse")  # G.I. og G.M.
+# Cache de relevante punktinformationstyper for kanonisk_ident,
+# så de ikke skal slås op ved hvert kald af funktionen
+pit_landsnr = firedb.hent_punktinformationtype("IDENT:landsnr")
+pit_gnssnavn = firedb.hent_punktinformationtype("IDENT:GNSS")
+pit_gi_gs_gm = firedb.hent_punktinformationtype("IDENT:diverse")
+
+
+def kanonisk_ident(uuid: str) -> str:
```

Comment:
Jeg synes den her egner sig bedre som en property på `Punkt`. Det er nyttigt mere end et sted at trække det bedste ident ud til et punkt. Endnu et sted det giver mening at cache værdien.

Reply:
Ja - property eller API-funktion. Jeg opretter et issue desangående (#155), til senere brug - men ville foretrække at det ikke forsinker dette PR yderligere: @majbrittws har brug for det